Tools/Scripts/webkitpy/benchmark_runner/data/plans/jetstream2.1.plan

```diff
@@ -0,0 +1,7 @@
+{
+    "timeout": 1200,
+    "count": 5,
+    "svn_source": "https://svn.webkit.org/repository/webkit/trunk/PerformanceTests/JetStream2/",
```

Comment:
Does this contain the 2.1 change?

Reply:
no 2.1 hasn't landed yet

Reply:
it would be TOT except it's still using svn so it won't be